examples/beam_slic/job.json

```diff
@@ -1,12 +1,14 @@
 {
     "run_number": 10666,
-    "nevents": 250000,
+    "nevents": 250,
     "detector": "HPS-PhysicsRun2019-v2-4pt5",
     "input_files": {
-        "/sdf/group/hps/mc/4pt55GeV/beam/20um120nA/beam_4pt55_20um120uA_1.stdhep": "beam.stdhep"
+        "/sdf/group/hps/mc/4pt55GeV/beam/20um120nA/beam_4pt55_20um120nA_60.stdhep": "beam_1.stdhep",
```

Comment:
@sarahgaiser Here is an input file in an example with a path on sdf.

Reply:
outdated commit